configuration/exodus-pipeline.yaml

```diff
@@ -260,9 +283,26 @@ Resources:
                   - Name: BuildArtifact
                 RunOrder: 2
           - !Ref AWS::NoValue
+        - !If
+          - IsProd
+          - Name: Upload
```

Comment:
I think we should consolidate uploading here (switch to !If NotDev and delete Upload from Promote stage).